cpp/src/arrow/compute/exec/asof_join_benchmark.cc

```diff
@@ -49,6 +49,17 @@ static Result<TableStats> MakeTable(const TableGenerationProperties& properties)
   return Result<TableStats>({table, rows, rows * row_size});
 }
 
+static ExecNode* MakeTableSourceNode(std::shared_ptr<arrow::compute::ExecPlan> plan,
```

Comment:
Can you add a comment of some kind explaining why you chose to use "table" directly instead of "table_source"?